packages/support/resources/views/components/button/index.blade.php

```diff
@@ -221,8 +221,14 @@
 
             @if ($hasLoadingIndicator)
                 <x-filament::loading-indicator
-                    wire:loading.delay=""
-                    :wire:target="$loadingIndicatorTarget"
+                    :attributes="
+                        \Filament\Support\prepare_inherited_attributes(
+                            new \Illuminate\View\ComponentAttributeBag([
+                                'wire:loading.delay.' . config('filament.wire_loading_delay', 'default') => '',
+                                'wire:target' => $loadingIndicatorTarget,
+                            ])
+                        )
+                    "
                     :class="$iconClasses"
```

Comment:
If you're using `:attributes`, please make sure to use `->class()` there instead of passing it separately as `:class`.

Reply:
Hey @zepfietje. That was my initial approach but I changed it to be consistent with this: https://github.com/filamentphp/filament/blob/865084713c11cb1f079a46f7e950438337520b38/packages/support/resources/views/components/input/wrapper.blade.php#L118.

No problems, will update it.

Reply:
Oh that case was probably missed by someone during a refactor or so.
Could you update that one too?

Reply:
@zepfietje Updated

Reply:
Thanks